tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java

```diff
@@ -558,10 +560,6 @@ private void writeEmptyFile(String id, Path dir, Collection<String> aliasTargets
         if (aliasTargets.contains(id)) {
             icuData.setFileComment("generated alias target");
             icuData.add(RB_EMPTY_ALIAS, "");
-        } else {
```

Comment:
Please keep the else and the comment and just delete the setVersion() line.

If it bothers you that there is nothing being done, you could add another comment line like
```
// No need to add data: Just write a resource bundle with an empty top-level table.
```

Reply:
Done.